fedimint-server/src/db.rs

```diff
@@ -31,6 +32,29 @@ impl std::fmt::Display for DbKeyPrefix {
     }
 }
 
+#[derive(Clone, Debug, Encodable, Decodable, Serialize)]
+pub struct SessionIndexKey;
+
+impl_db_record!(
+    key = SessionIndexKey,
+    value = u64,
+    db_prefix = DbKeyPrefix::SessionIndex,
+);
+
+#[derive(Clone, Debug, Eq, PartialEq, Encodable, Decodable, Serialize, Ord, PartialOrd)]
+pub struct AcceptedIndex(pub u64);
```

Comment:
Would love to see docs here as well. I can figure out the code if I know what data means, but if I don't know what data means, I don't even know my own name.

Reply:
Generally the broadcast code is not final yet. Expect some refactors and changes over the coming weeks. Will do docs when design is more stable.